lib/matplotlib/tests/test_axes.py

```diff
@@ -786,6 +786,23 @@ def test_hexbin_log():
     plt.colorbar(h)
 
 
+@image_comparison(['hexbin_linear.png'], style='mpl20')
+def test_hexbin_linear():
+    # Issue #21165 (and also test linear scaled)
```

Comment:
Don't reference a number; describe what this does, or don't include the comment at all.

Reply:
ok, I will solve it, thanks